base/workflow/R/run.write.configs.R

```diff
@@ -167,7 +180,8 @@ run.write.configs <- function(settings, write = TRUE, ens.sample.method = "unifo
                                                           ensemble.samples = ensemble.samples, 
                                                           settings = settings,
                                                           model = model, 
-                                                          write.to.db = write)
+                                                          write.to.db = write,
+                                                          input_design= input_design)
```

Comment:
Nit 1: spaces on both sides of the equal sign
Nit 2: function calls are a little easier to read if the arguments are written in the same order as the formal signature (R doesn't care at all, this is just for the humans)
```suggestion
                                                          input_design = input_design,
                                                          write.to.db = write)
```